quinn-proto/src/crypto.rs

```diff
@@ -40,8 +40,6 @@ pub trait Session: Send + Sized {
     type HmacKey: HmacKey;
     /// Pseudo random key used for generating AEAD key
     type PseudoRandomKey: PseudoRandomKey;
```

Comment:
On review I think this would be a lot clearer if the name described the application, since we're not trying to define a general-purpose PRK abstraction here.
```suggestion
    /// Key used to generate one-time-use handshake token keys
    type HandshakeTokenKey: HandshakeTokenKey;
```
and similar for the trait definition, of course.